en/uavcan/node_firmware.md

```diff
@@ -1,25 +1,3 @@
-# UAVCAN Firmware Upgrading
-
-PX4 will automatically upgrade firmware on UAVCAN nodes if the appropriate firmware is supplied.
-The firmware build process is manufacturer dependent (instructions for different firmware linked below).
-
-:::warning
-UAVCAN devices typically ship with appropriate firmware preinstalled. 
-These instructions are primarily needed when developing UAVCAN devices.
-:::
-
-## Flashing/Updating a UAVCAN Binary
-
-To flash the UAVCAN device, all you need to do is copy the UAVCAN firmware binary into the root directory of the flight controller’s SD card and reboot.
-
-After restarting the flight controller the UAVCAN device will automatically be updated with new firmware.
-If successful the firmware binary will be removed from the root directory and there will be a file named **XX.bin** in the **/ufw** directory of the SD card.
-
-:::note
-UAVCAN firmware is usually created as part of the build process.
-Commonly the firmware image is built in **build/_name_of_your_uavcan_device_**,  where _name_of_your_uavcan_device_ is specific to the device you are flashing (e.g. **build/ark_can-flow_default**), and will be named with the format **XX-X.X.XXXXXXXX.uavcan.bin**.
-:::
-
```

Comment:
This doc seems to have been partially stripped. What is the plan here?

The idea would be to remove all mentions of UAVCAN from here. Either make this stuff cyphal or dronecan specific, or push it into those docs sections.

Reply:
@coderkalyan Same story for this. Some of it moved out, but is the story on this page still consistent (a question, not a statement)